server/sessionstore.go

```diff
@@ -63,6 +63,7 @@ func (ss *SessionStore) NewSession(conn interface{}, sid string) (*Session, int)
 		s.send = make(chan interface{}, 256) // buffered
 		s.stop = make(chan interface{}, 1)   // Buffered by 1 just to make it non-blocking
 		s.detach = make(chan string, 64)     // buffered
+		s.remoteSubs = make(map[string]*RemoteSubscription)
```

Comment:
It would be a good idea to initialize`s.remoteSubs` only when `globals.cluster` is not `nil`. Otherwise it's a waste of memory.

Reply:
Done.